examples/default/main.tf

```diff
@@ -71,14 +71,14 @@ module "vpes" {
   cloud_services       = var.cloud_services
   cloud_service_by_crn = var.cloud_service_by_crn
   service_endpoints    = var.service_endpoints
-  #  Wait 30secs after security group is destroyed before destroying VPE to workaround timing issue which can produce “Target not found” error on destroy
```

Comment:
whats the reasoning for increasing this timelimit to 120secs? Does it solve the "target not found" issue?

Reply:
with 30s wait the failures were random, sometimes it destroyed successfully and sometimes it failed with "target not found". After increasing the sleep time to 120s didn't face that issue of random failures.